support-frontend/app/views/main.scala.html

```diff
@@ -148,7 +148,7 @@
     </script>
     <script
       id="polyfill-script"
-      src="https://polyfill.io/v2/polyfill.min.js?features=default,Array.prototype.find,fetch,Array.prototype.includes,Number.isInteger,Object.values,Object.entries"
+      src="https://polyfill.io/v3/polyfill.min.js?features=default,Array.prototype.find,fetch,Array.prototype.includes,Number.isInteger,Object.values,Object.entries"
       onload="polyfillOnLoad()"
```

Comment:
I wonder if, instead of polyfillOnLoad you could use the Polyfill Callback functionality `https://polyfill.io/v3/polyfill.min.js?callback=polyfillOnLoad`? A bit of a shot in the dark but might be more reliable...

Reply:
that's a great suggestion - didn't know about that callback param!